docs/en/operations/query-cache.md

```diff
@@ -31,6 +31,10 @@ This reduces maintenance effort and avoids redundancy.
 
 ## Configuration Settings and Usage
 
+:::note
```

Comment:
Very generally speaking, various settings are referred to in various places of the ClickHouse documentation. Should we really add a note to each of these places that it is not possible to edit configuration files in ClickHouse Cloud? In my view, such a note should appear ideally once somewhere in the CH Cloud documentation (https://clickhouse.com/docs/en/cloud/overview).

Reply:
We actually do state that globally for cloud here: https://clickhouse.com/docs/en/manage/settings

This PR was requested by @alexey-milovidov so I'll leave it up to him. My thoughts are while it is repetitive to include this note, for common settings that users have to configure to be able to use a feature it's fine to let them know to use query level settings or settings profiles.

Reply:
Okay, fine with me then.